lib/timer.js

```diff
@@ -0,0 +1,11 @@
+var gauge = require('./gauge');
+var types = require('./types');
+
+module.exports = function(metricsClients) {
```

Comment:
~~Potentially should be passed `metricsClient` instead of `metricsClients`, if my earlier comment in `lib/cpu.js` is accurate.~~

Edit: judging by `lib/timingMiddleware.js` below, it should probably be `metricsClients` everywhere.